client/data/authorizations/hooks.ts

```diff
@@ -14,34 +14,57 @@ import {
 	Authorizations,
 } from 'wcpay/types/authorizations';
 
-export const useAuthorizations = ( {}: Query ): Authorizations =>
-	useSelect( ( select ) => {
-		const {
-			getAuthorizations,
-			getAuthorizationsError,
-			isResolving,
-		} = select( STORE_NAME );
+export const useAuthorizations = ( {
+	paged,
+	// eslint-disable-next-line @typescript-eslint/naming-convention
+	per_page,
+	orderby,
```

Comment:
Same as ^, the `orderby` is defined in the `Query` type `client/transactions/declarations.d.ts`, so I think changing it would require changing it everywhere. Unless we want to rename it in the deconstruction `orderby: orderBy` but I don't think that's going to add any value?

Reply:
Yep, feel free to skip.